easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.2.19-GCC-6.4.0-2.28-LAPACK-3.7.0.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenBLAS'
+version = '0.2.19'
```

Comment:
Seems like they screwed something up with the OpenBLAS 0.2.20 release. Another quick 0.2.21 release is planned, probably worth waiting for this? See https://github.com/xianyi/OpenBLAS/issues/1258.

Thoughts @easybuilders/easybuild-easyconfigs-maintainers?